types/webpack/index.d.ts

```diff
@@ -664,6 +603,8 @@ declare namespace webpack {
             assetsSort?: string;
             /** Add information about cached (not built) modules */
             cached?: boolean;
+            /** Show cached assets (setting this to `false` only shows emitted files) */
+            cachedAssets?: true;
```

Comment:
Shouldn't this be `boolean` instead of `true`?

Reply:
FIxed in #21342